x-pack/plugins/uptime/public/lib/alert_types/translations.ts

```diff
@@ -49,3 +49,23 @@ Aging Certificates: {agingCommonNameAndDate}
     defaultMessage: 'Uptime TLS',
   }),
 };
+
+export const DurationAnomalyTranslations = {
+  defaultActionMessage: i18n.translate('xpack.uptime.alerts.durationAnomaly.defaultActionMessage', {
+    defaultMessage: `A {severity} level anomaly with score {severityScore} was detected at {anomalyStartTimestamp} on {monitor} response duration in Uptime at url {monitorUrl}.
```

Comment:
Reading the output, this first sentence is very awkward (I know, I wrote the original). WDYT of the following?

```suggestion
    defaultMessage: `Abnormal ({severity} level) response time detected on {monitor}({monitorUrl} at {anomalyStartTimestamp}. Anomaly severity score is {severityScore}.
```